### PR TITLE
Restructure claim token API

### PR DESCRIPTION
**Pull request by feritka79**
restructur token apis and add token recieve api

### CODE REVIEW

**Review by feritka79 — COMMENTED** [review]
Reviewed all comment

**Review by ceydae — CHANGES_REQUESTED** [review]
Enforce the use of IResponseTemplate for all routes.
Change the PR title and description

File: src/routes/token/add.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    const newToken = new Token({
+      address: token,
+      name: result[0],
+      symbol: result[1],
+      decimals: result[2],
+    });
+
+    await newToken.save();
+
+    return res.status(200).json({
```
Comment (ceydae) [inline src/routes/token/add.ts:56]: use 201 whenever something is being created.

**Review by ceydae — CHANGES_REQUESTED** [review]

File: src/swagger/token/get.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/**
+ * @swagger
+ * /token:
+ *   get:
+ *     summary: Receives tokens from the list of claimable tokens.
```
Comment (ceydae) [inline src/swagger/token/get.ts:5]: Returns the list of the claimable tokens

**Review by ceydae — CHANGES_REQUESTED** [review]

**Review by ceydae — APPROVED** [review]

**Pull request merged**